os/drivers/task_manager/task_manager_drv.c

```diff
@@ -92,6 +101,71 @@ static int taskmgr_task_init(pid_t pid)
 	return sig_sethandler(tcb, SIGTM_PAUSE, &act);
 }
 
+#if defined(HAVE_TASK_GROUP) && !defined(CONFIG_DISABLE_PTHREAD)
+static void taskmgr_pthread_group_detach(pid_t pthread_pid)
+{
+	FAR struct tcb_s *pthread_tcb;
+
+	pthread_tcb = sched_gettcb(pthread_pid);
```

Comment:
we need to check whether pthread_tcb is valid or not

Reply:
Done